src/js/post-911-gib-status/components/EnrollmentPeriod.jsx

```diff
@@ -1,35 +1,95 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Scroll from 'react-scroll';
 
 import InfoPair from './InfoPair';
 
+import { getScrollOptions } from '../../common/utils/helpers';
 import { formatDateShort } from '../utils/helpers';
 
+const scroller = Scroll.scroller;
+
 class EnrollmentPeriod extends React.Component {
+  constructor() {
+    super();
+    this.scrollToEnrollment = this.scrollToEnrollment.bind(this);
+    this.toggleHistory = this.toggleHistory.bind(this);
+    this.state = { historyExpanded: false };
+  }
+
+  scrollToEnrollment() {
+    const options = getScrollOptions({ delay: 2 });
+    scroller.scrollTo(`collapsible-${this.props.id}`, options);
+  }
+
+  toggleHistory() {
+    const shouldExpand = !this.state.historyExpanded;
+    this.setState({ historyExpanded: !this.state.historyExpanded });
+    if (shouldExpand) {
+      this.scrollToEnrollment();
+    }
+  }
+
   render() {
-    const { enrollment } = this.props;
+    const { enrollment, id } = this.props;
+    const amendments = enrollment.amendments || [];
+
+    // TODO: should fullTimeHours be used anywhere?
```

Comment:
@annekainicUSDS Good question, that I unfortunately don't have a concrete answer to. If it's different from on/off-campus hours, then I would guess we do _not_ have to use it -- eBenefits does not. 

@rroueche can we add this question to our next check-in? I don't think it's worth an e-mail at this stage, but would like to confirm that we don't need to use that piece of data.